pages/common/cowsay.md

```diff
@@ -1,24 +1,28 @@
 # cowsay
 
-> Generate an ASCII character (by default a cow) saying or thinking something.
+> Print an ASCII friend (by default a cow) saying or thinking something.
 > More information: <https://github.com/tnalpgge/rank-amateur-cowsay>.
 
-- Print an ASCII cow saying "Hello world":
+- Print an ASCII cow saying "Hello World":
 
-`cowsay "Hello world"`
+`cowsay "{{Hello World}}"`
 
 - Read text from stdin for the balloon:
```

Comment:
```suggestion
- Print an ASCII cow saying text from stdin:
```

Reply:
This comment was left from the original page, however it sounds a little man pagey, so i'll commit that